pwndbg/elf.py

```diff
@@ -30,16 +26,21 @@
 import pwndbg.proc
 import pwndbg.stack
 
+from imp import reload
```

Comment:
Not sure if that is the proper way for compatibility? [An online cheatsheet](http://python-future.org/compatible_idioms.html) suggests it.

However, in our own reload command, we use [reload from either `builtins` or from `imp`](https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/reload.py#L20-L23). Is there any reason to do so?